src/c4/ifpstream.cc

```diff
@@ -0,0 +1,69 @@
+//----------------------------------*-C++-*-----------------------------------//
+/*!
+ * \file   c4/ifpstream.cc
+ * \author Mathew Cleveland
+ * \date   Feb. 2020
+ * \brief  Define methods of class ifpstream
+ * \note   Copyright (C) 2018-2020 Triad National Security, LLC.
+ *         All rights reserved. */
+//----------------------------------------------------------------------------//
```

Comment:
Can you add a note here (or maybe in the hh file) to note this is the counter-part to ofpstream and that the logic is similar.

Reply:
Yap. fixed.